app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  after_create :post_counter
+  after_destroy :post_counter
+
+  def recent_comments
+    comments.last(5).reverse
+  end
+
+  def post_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `post_counter` method every time a new post has been saved to the database, then make the `post_counter` method private